recipes/psdm_qs_cli/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "psdm_qs_cli" %}
+{% set version = "0.2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: v{{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
Same as above, if we switch the package upstream to use the prefix at the tag we also need to change it here.